disk/datadog_checks/disk/data/conf.yaml.default

```diff
@@ -16,7 +16,8 @@ init_config:
     ## @param mount_point_global_blacklist - list of strings - optional
     ## Instruct the check to always add these patterns to `mount_point_blacklist`.
     #
-    # mount_point_global_blacklist: []
+    # mount_point_global_blacklist:
+    #   - /proc/sys/fs/binfmt_misc$
```

Comment:
Should we add a comment that overriding this value will introduce possible problems on systems using `systemd`?

Reply:
Hmm, dunno if a comment in description for each would scale long term. WDYT?